src/python/pants/backend/experimental/python/lockfile_metadata_test.py

```diff
@@ -55,14 +55,14 @@ def line_by_line(b: bytes) -> list[bytes]:
     assert line_by_line(result) == line_by_line(expected)
 
 
-_requirements = ["flake8-pantsbuild>=2.0,<3", "flake8-2020>=1.6.0,<1.7.0"]
-
-
 @pytest.mark.parametrize(
     "requirements,expected",
     [
         ([], "c8e8d0a6d6ec36bee3942091046d81d86e3b83b143b37a7cc714e2d022bf4f85"),
-        (_requirements, "66327c52225d2f798ffad7f092bf1b51da8a66777f3ebf654e2444d7eb1429f4"),
+        (
```

Comment:
imo this test has never been particularly useful, since it tests the specific implementation of the hash, and it's not really verifiable that it is right. What matters is that it behaves predictably with semantics that lockfile consumers can rely upon.

At this point though, there are useful semantics that we can test.

* Test that `digest([A, B]) == digest([B, A])`
* Test that `digest([]) != digest([A])`
* Test that `digest([A, B]) != digest([A, B, C])`
* Test that `digest([A, A]) == digest([A])` for identical `A` :)

I think that just about covers it.